src/frontend/templates/header.html

```diff
@@ -46,6 +46,13 @@
 
 <body>
     <header>
+        {{ if $.frontendMessage }}
+        <div class="navbar">
+            <div class="container d-flex justify-content-center">
+                <div class="h-free-shipping">{{ $.frontendMessage }}</div>
+            </div>
+        </div>
+        {{ end }}
```

Comment:
We don’t need to sanitize `frontendMessage`. HTML injection is prevented by the [Golang html/template](https://pkg.go.dev/html/template).